src/Crypto/Cryptograph.php

```diff
@@ -15,11 +15,11 @@ class Cryptograph
     /**
      * Constructor.
      *
-     * @param int $length
+     * @param int|null $length
      */
-    public function __construct($length = null)
+    public function __construct(int $length = null)
     {
-        $this->length = $length ?: 0;
+        $this->length = $length ?? 0;
```

Comment:
Yeah, i actually need to redo the whole padding bit. The purpose is that you can make all messages a similar length, the current implementation just adds length i think. I left it in for now because it does fulfill a purpose which is add the mandatory 2 bytes of padding when yet set the padding to 0.